README.md

```diff
@@ -111,7 +111,7 @@ In Illinois, Indiana, Iowa, Kansas, Michigan, Minnesota, Missouri, Nebraska, Nor
   - [Startup Iowa](http://www.startupiowachat.com/)
   - [WordPress Des Moines](http://wpdsm.org/)(http://wordpressdsm.herokuapp.com/)
 - IL - **In Illinois there are several channels:**
-  - [Chicago Tech](http://www.chicagotechslack.com/)
+  - [Chicago Tech](http://www.chitechdiversity.com/)
```

Comment:
We need to remove this entry completely because there is already an active Chicago Tech Diversity Initiative entry on line 117.